### PR TITLE
Updated Header to be responsive

### PR DESCRIPTION
**Pull request by sungod12**
-Made changes in css to be responsive in almost all latest and some old devices.
-Few changes were in .css files ,and some of them were in tailwind css.
-Changed login and signup buttons to appear inside a menu in small and medium devices.
Please review @euhidaman and let me know.

### CODE REVIEW

File: src/components/SignUpButton.css [hunk 1]
```diff
@@ -4,3 +4,9 @@
   margin-left: 50px;
   margin-top: -15px;
 }
+@media screen and (max-width: 767px) {
```
Comment (sungod12) [inline src/components/SignUpButton.css:7]: removed margin left and right in smaller devices to appear properly in menu

**Comment by sungod12** [comment]
Could you change the label to hacktoberfest-acccepted label?.

**Comment by euhidaman** [comment]
Well done !!
And, thank you for contributing !!☺☺

**Comment by sungod12** [comment]
Thanks bro. Yeah did you check the changes ?.

**Comment by euhidaman** [comment]
I have reviewed and merged the changes.
Well done !!

**Comment by sungod12** [comment]
> I have reviewed and merged the changes. Well done !!

okay. Just wanted to know if it displays correctly in your device or not.

**Pull request merged**